translations/base-fr.yaml

```diff
@@ -510,37 +511,37 @@ buildings:
             description: Colorie les deux formes de gauche avec la couleur de droite.
         quad:
             name: Peintre (Quadruple)
-            description: Permet de colorier chaque quadrant d'une forme avec une couleur différente.
+            description: Permet de colorier chaque quadrant d’une forme avec une couleur différente.
         mirrored:
             name: *painter
             description: *painter_desc
 
     trash:
         default:
             name: &trash Poubelle
-            description: Accepte des formes de n'importe quel côté et les détruit... pour toujours.
+            description: Accepte des formes de n’importe quel côté et les détruit… pour toujours.
 
         storage:
             name: Stockage
-            description: Stocke les formes en trop jusqu'à une certaine capacité. Peut être utilisé comme tampon.
+            description: Stocke les formes en trop jusqu…à une certaine capacité. Peut être utilisé comme tampon.
```

Comment:
I think you wanted to make an apostrophe here

Reply:
Yes. Bad copy/paste.